cli/cmd/context/use.go

```diff
@@ -31,22 +28,8 @@ func useCommand() *cobra.Command {
 		Short: "Set the default context",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runUse(args[0])
+			mobycli.Exec(cmd.Root())
```

Comment:
@nicks did you check that the `mobycli` can `context use` a context that's created by compose-cli?

Reply:
as far as i can tell, they're interoperable -- they read and write from the same metadata tree under `~/.docker/contexts/meta`. Here's the sequence of commands I ran:

```
nick@grumpy:~/src/compose-cli$ com.docker.cli context create ecs ecs-test --from-env 
unknown flag: --from-env
See 'docker context create --help'.
nick@grumpy:~/src/compose-cli$ go run ./cli/main.go context create ecs ecs-test --from-env 
Successfully created ecs context "ecs-test"
nick@grumpy:~/src/compose-cli$ com.docker.cli context list
NAME            DESCRIPTION                               DOCKER ENDPOINT                                 KUBERNETES ENDPOINT                                 ORCHESTRATOR
default *       Current DOCKER_HOST based configuration   unix:///var/run/docker.sock                     https://kubernetes.docker.internal:6443 (default)   swarm
desktop-linux                                             unix:///home/nick/.docker/desktop/docker.sock                                                       
ecs-test        credentials read from environment                                                                                                             
nick@grumpy:~/src/compose-cli$ com.docker.cli context use ecs-test
ecs-test
Current context is now "ecs-test"
nick@grumpy:~/src/compose-cli$ com.docker.cli context use default
default
Current context is now "default"
nick@grumpy:~/src/compose-cli$ com.docker.cli context rm ecs-test
ecs-test
nick@grumpy:~/src/compose-cli$ com.docker.cli context list
NAME            DESCRIPTION                               DOCKER ENDPOINT                                 KUBERNETES ENDPOINT                                 ORCHESTRATOR
default *       Current DOCKER_HOST based configuration   unix:///var/run/docker.sock                     https://kubernetes.docker.internal:6443 (default)   swarm
desktop-linux                                             unix:///home/nick/.docker/desktop/docker.sock
```

do you think it makes sense to add an integration test for it?

Reply:
I think it might make sense yes, I know that the context management code was changed recently in docker/cli, hopefully the change still handles contexts with additional metadata